packages/iris-grid/src/IrisGridTreeTableModel.ts

```diff
@@ -176,15 +176,16 @@ class IrisGridTreeTableModel extends IrisGridTableModelTemplate<
           this.viewportData.rows[r - this.viewportData.offset];
         assertNotNull(intersection.startColumn);
         assertNotNull(intersection.endColumn);
-        for (
-          let c = intersection.startColumn;
-          c <= intersection.endColumn;
-          c += 1
-        ) {
-          assertNotNull(formatValue);
-          resultRow.push(
-            formatValue(viewportRow.data.get(c)?.value, this.columns[c])
-          );
+        if (formatValue !== undefined) {
```

Comment:
Usually preferred to use loose equality `!= null` here since it will catch `null` and `undefined` scenarios.

Reply:
Unless of course you have a reason for explicitly catching `undefined` and ignoring null.

Reply:
Don't have a specific reason for explicitly catching undefined and ignoring null, actually in my first commit I had it set to null, so can definitely change it back.

Reply:
I had only changed it to undefined to be more in alignment with your comment, but yeah I think it is a better choice to use null here